exercises/bank-account/src/main/scala/Bank.scala

```diff
@@ -0,0 +1,13 @@
+trait BankAccount {
+
+  def closeAccount(): Unit
+
+  def getBalance: Option[Int]
+
+  def incrementBalance(increment: Int): Option[Int]
+}
+
+object Bank {
```

Comment:
Why don't you want to use the companion object anymore?
Same for the file name. AFAIK the file name is always the same as the class/object to be implemented?